### ISSUE TITLE
Issue with angular inlineCritical optimization

### ISSUE THREAD

**Issue by Alddar** [scttcper/ngx-toastr#900]
Hello.
Due to a issue in https://github.com/reworkcss/css/issues/154, it is impossible to use Angular's inlineCritical optimization (https://github.com/angular/angular-cli/issues/20760).

The ngrx-toastr code that causes this issue is located here https://github.com/scttcper/ngx-toastr/blob/f2c4c424bc28152012519d7a52c5ebee9c84fcae/src/lib/toastr.css#L114-L129

Would it be possible to try a different approach until the https://github.com/reworkcss/css library is fixed? By the looks of things there is a https://github.com/reworkcss/css/pull/157 that fixes the issue, but I have no idea if or when it will be merged.

Thanks.

**Comment by loetscher** [comment]
Same issue here. As we use `ngx-toastr` within a frontend library, it took me some time to figure out what exactly the problem is.
I got the same error as mentioned in https://github.com/nrwl/nx/issues/4381:
```
× Index html generation failed.
```
The workaround mentioned in https://github.com/nrwl/nx/issues/4381#issuecomment-849431451 "fixed" the issue temporarly.

**Comment by loetscher** [comment]
I tried to provide a simple sandbox to reproduce this problem.
It looks like our problem is an interaction between ngx-toastr <-> ag-grid.

I created a small sample project https://github.com/loetscher/ngx-toastr-issue-ninehundred (angular starter with these two libraries)

The prod build of this project fails, but if I:
- remove ag-grid `@include ag-theme-balham`
- move import statement for ngx-toastr at the end
- remove import satement for for ngx-toastr 
- apply changes in https://github.com/scttcper/ngx-toastr/pull/901
everything works fine.

See https://github.com/loetscher/ngx-toastr-issue-ninehundred/blob/main/src/styles.scss

It looks like there is a problem in combination with those two projects.
